packages/regl-worldview/src/commands/GLText.js

```diff
@@ -320,11 +358,15 @@ function makeTextCommand() {
           marker.colors?.[1] || (command.autoBackgroundColor && isColorDark(fgColor) ? BG_COLOR_LIGHT : BG_COLOR_DARK);
         const hlColor = marker?.highlightColor || { r: 1, b: 0, g: 1, a: 1 };
 
+        if (!marker.billboard && command.scaleInvariant) {
+          console.warn("Scale invariant option is only supported for billboard markers");
```

Comment:
I'd be more aggressive and just throw an error, so that we're really sure people don't do this:

```suggestion
          throw new Error("Scale invariant option is only supported for billboard markers");
```

Reply:
@janpaul123 I agree about having a more aggressive approach here. While the code is handling this case and just ignoring scale invariance for non billboards, a user will only be aware of this when looking at the debugger's console. If we thrown an error instead, the problem is much more visible and easier to fix.
What do you think @troygibb?

Reply:
sounds good to me!

Reply:
Probably doesn't matter, but is it reasonable to enforce this strucutrally? Maybe signal `billboard` with a nullable object containing billboard-relevant options like scale invariance?

Reply:
Ah that's a great idea, e.g. ```js
type Props = {|
  …other stuff…
  billboard: boolean | {| scaleInvariantFontSize?: number |}
|};
```

Because it probably never makes sense to use scale invariance without billboards.

Reply:
We decided not to add new options to markers and remove the warning message instead.